benchmarks/benchmark-partial-render-table/Makefile

```
@@ -2,7 +2,8 @@
 
 export NAME=Partial Render Table
 
-SHELL=/bin/bash -o pipefail
+SHELL=/usr/bin/env bash -o pipefail
+TIME=$(shell whereis time | cut -d ' ' -f 2)
```

```suggestion
TIME=$(shell which time)
```
Or you can use `env time` or simply `\time` instead of `/usr/bin/time` below.

---

Thanks!